ddtrace/appsec/_iast/_taint_tracking/aspects.py

```diff
@@ -540,6 +540,86 @@ def format_value_aspect(
         return new_text
 
 
+def ospathjoin_aspect(*args, **kwargs):
+    try:
+        return _aspect_ospathjoin(*args, **kwargs)
+    except Exception as e:
+        iast_taint_log_error("IAST propagation error. ospathjoin_aspect. {}".format(e))
+        import os.path
+
+        return os.path.join(*args, **kwargs)
```

Comment:
Not sure about this approach, what if the `os.path.join` is a mock? I think we should pass the original function here